android/ANDROID.md

```diff
@@ -0,0 +1,124 @@
+# AWS CRT Android
+
+This document provides information about building and using the AWS CRT Java with Android.
+
+If you have any issues or feature requests, please file an issue or pull request.
+
+API documentation: https://awslabs.github.io/aws-crt-java/
+
+This SDK is built on the AWS Common Runtime, a collection of libraries
+([aws-c-common](https://github.com/awslabs/aws-c-common),
+[aws-c-io](https://github.com/awslabs/aws-c-io),
+[aws-c-mqtt](https://github.com/awslabs/aws-c-mqtt),
+[aws-c-http](https://github.com/awslabs/aws-c-http),
+[aws-c-cal](https://github.com/awslabs/aws-c-cal),
+[aws-c-auth](https://github.com/awslabs/aws-c-auth),
+[s2n](https://github.com/awslabs/s2n)...) written in C to be
+cross-platform, high-performance, secure, and reliable.
+
+*__Jump To:__*
+
+* [Installation](#installation)
+  * [Minimum requirements](#minimum-requirements)
+  * [Build and install CRT from source](#build-and-install-crt-from-source)
+* [Consuming AWS CRT Android](#consuming-aws-crt-android)
+  * [Consuming from Maven](#consuming-from-maven)
+  * [Consuming from locally installed](#consuming-from-locally-installed)
+
+## Installation
+
+### Minimum requirements
+* Java 11+ ([Download and Install Java](https://www.java.com/en/download/help/download_options.html))
+  * [Set JAVA_HOME](#set-java_home)
+* Gradle 7.4.2 ([Download and Install Gradle](https://gradle.org/install/))
+* Android SDK 24 ([Doanload SDK Manager](https://developer.android.com/tools/releases/platform-tools#downloads))
+  * [Set ANDROID_HOME](#set-android_home)
+* Android NDK ([Download and install Android NDK](https://developer.android.com/ndk/downloads))
+
+### Build and install CRT from source
+Supports API 24 or newer.
+
+``` sh
+# Create a workspace directory to hold all the SDK files
+mkdir sdk-workspace
+cd sdk-workspace
+# Clone the CRT repository
+# (Use the latest version of the CRT here instead of "v0.27.6)
+git clone --branch v0.27.6 --recurse-submodules https://github.com/awslabs/aws-crt-java.git
+# Compile and install the CRT for Android
+./gradlew :android:crt:build
+# Install CRT locally
+./gradlew :android:crt:publishToMavenLocal
+```
+
+## Consuming AWS CRT Android
+
+### Consuming from Maven
+(#consuming-from-locally-installed)
```

Comment:
is the `(#consuming-from-locally-installed)` tag here necessary?

Reply:
I don't know why this is here =P